src/AddressRegistry.Api.BackOffice/Infrastructure/Startup.cs

```diff
@@ -3,18 +3,18 @@ namespace AddressRegistry.Api.BackOffice.Infrastructure
     using System;
     using System.Linq;
     using System.Reflection;
+    using Asp.Versioning.ApiExplorer;
     using Autofac;
     using Autofac.Extensions.DependencyInjection;
     using Be.Vlaanderen.Basisregisters.Auth.AcmIdm;
     using Be.Vlaanderen.Basisregisters.AggregateSource.SqlStreamStore;
     using Be.Vlaanderen.Basisregisters.Api;
     using Be.Vlaanderen.Basisregisters.CommandHandling.Idempotency;
-    using Be.Vlaanderen.Basisregisters.DataDog.Tracing.Microsoft;
+    using Be.Vlaanderen.Basisregisters.DataDog.Tracing.Autofac;
```

Comment:
I'm missing this change
https://github.com/Informatievlaanderen/building-registry/blob/main/src/BuildingRegistry.Api.BackOffice/Infrastructure/Startup.cs#L94

Reply:
Then you'll need to add https://github.com/Informatievlaanderen/building-registry/blob/main/src/BuildingRegistry.Api.BackOffice/Infrastructure/Modules/FluentValidationModule.cs